charmcraft/application/commands/store.py

```diff
@@ -1225,7 +1225,7 @@ class CreateLibCommand(CharmcraftCommand):
     help_msg = "Create a charm library"
     overview = textwrap.dedent(
         """
-        Create a charm library.
+        Create a Charmhub-hosted charm library.
```

Comment:
I'm wondering if we could make the pattern for these messages read more naturally. How would this sound to you?

```suggestion
        Create a charm library on Charmhub.
```

Reply:
I'm not committed to the 'Charmhub-hosted' phrasing everywhere, but in this particular case it seems like the new wording isn't accurate -- it creates a charm library locally, but it's not created on Charmhub until publishing.

Here I'd like it to be my initial suggestion, or perhaps:
```suggestion
        Create a Charmhub-hosted library.
```

I'll post rewording suggestions for the other command help, lmk what you think.

Reply:
This one also LGTM.